Add redirect notice for documents moved from mimir to helm-chart docs (#3954)
grafana/mimir: docs/sources/mimir/operators-guide/deploy-grafana-mimir/getting-started-helm-charts/_index.md

---
aliases:
  - /docs/mimir/latest/operators-guide/deploying-grafana-mimir/getting-started-helm-charts/
description: Learn how to get started with Grafana Mimir using the Helm chart.
menuTitle: Getting started using the Helm chart
title: Getting started with Grafana Mimir using the Helm chart
weight: 25
---

> Contents have been moved to the [documentation for the Grafana Mimir Helm chart](/docs/helm-charts/mimir-distributed/latest/).

Reviewer: ```suggestion
This documentation has moved to the [Grafana Mimir Helm chart]({{< relref="/docs/helm-charts/mimir-distributed/latest/" >}}).
```

Reviewer: I think we need a relref here, oder?

Author: Relrefs don't start at an absolute path like `/docs` they would start with `./` or `../` . Also I'm not convinced relref could point to `latest`. I'm changing the wording though.

Author: I've updated it , although "for" looks a bit weird, maybe "in" the Grafana Mimir Helm chart documentation ?

Reviewer: Yes, please use "in".

Reviewer: > Relrefs don't start at an absolute path like `/docs` they would start with `./` or `../` . Also I'm not convinced relref could point to `latest`. I'm changing the wording though.

I tested this against `next` locally.
